packages/manifold/src/Manifold.ts

```diff
@@ -202,12 +202,9 @@ export const getFees = async (
         (isExclusive ? parseEther('0.00069') : parseEther('0.0005')) *
         quantityToMint
 
-      // calculate action fee
-      const mintPrice = data.mintPrice
-      let actionFee = 0n
-      if (mintPrice && typeof mintPrice === 'number') {
-        actionFee = parseEther(mintPrice.toString()) * quantityToMint
-      }
+      const { value, currency } = data.publicData.mintPrice
+      const mintPrice = currency === 'ETH' ? BigInt(value) : 0n
```

Comment:
using the example response from their docs:

```
{
	"id": 1851414768,
	"creator": {
		"id": 1863852272,
		"image": null,
		"name": "yungwknd",
		"twitterUrl": "richerd",
		"address": "0x6140f00e4ff3936702e68744f2b5978885464cbb"
	},
	"slug": "ferrari",
	"publicData": {
		"name": "scuderia ferrari",
		"image": "https://assets.manifold.xyz/image/upload/c_limit,q_80,w_1024/bc68f488caf37ea2a5e235a7818c09b66f8613bedae1e96b8fce41610ace992f.jpg",
		"endDate": "2022-10-24T21:00:00.000Z",
		"tokenUrl": "https://assets.manifold.xyz/video/upload/c_limit,q_80,w_1024/e1d062fb91c800ec7f15a2523c6d11c0c3a189d1bb7da3b4713c30aac8fa8a52.mp4",
		"claimType": "ERC1155",
		"isPayable": true,
		"startDate": null,
		"audienceId": null,
		"claimIndex": 2,
		"description": "celebrating leclerc and in memory of sainz\n\n🐴🏎",
		"extensionAddress": "0x4e32004d8b81847a670b4a1778ace4dcf2bba01e",
		"mediaIsLandscape": true,
		"creatorContractAddress": "0xefe6a6032fd27b7b7615fb0d0e08fb3e49db53b8"
	},
	"appId": 2537426615,
	"mintPrice": "0.0069",
	"isOpenEdition": true
}
```

mint price appears to be a stringified float

`BigInt( "0.0069")` will throw an error